### PR TITLE
translate into Chinese (simplified)  under install/SetupWizard/ 

### PR DESCRIPTION
**Pull request by node**
Please help to review @oleg-nenashev @LinuxSuRen .

### CODE REVIEW

File: core/src/main/resources/jenkins/install/pluginSetupWizard_zh_CN.properties [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+installWizard_welcomePanel_banner=\u81ea\u5b9a\u4e49jenkins
+installWizard_welcomePanel_message=\u63d2\u4ef6\u901a\u8fc7\u9644\u52a0\u7279\u6027\u6765\u6269\u5c55jenkins\u4ee5\u6ee1\u8db3\u4e0d\u540c\u7684\u9700\u6c42\u3002
+installWizard_welcomePanel_recommendedActionTitle=\u5b89\u88c5\u63a8\u8350\u7684\u63d2\u4ef6
+installWizard_welcomePanel_recommendedActionDetails=\u5b89\u88c5jenkins\u793e\u533a\u63a8\u8350\u7684\u63d2\u4ef6\u3002
+installWizard_welcomePanel_customizeActionTitle=\u9009\u62e9\u63d2\u4ef6\u6765\u5b89\u88c5
+installWizard_welcomePanel_customizeActionDetails=\u9009\u62e9\u5e76\u5b89\u88c5\u6700\u9002\u5408\u7684\u63d2\u4ef6\u3002
+installWizard_jenkinsVersionTitle=Jenkins 
+installWizard_offline_title=\u79bb\u7ebf
+installWizard_offline_message=\u8be5jenkins\u5b9e\u4f8b\u4f3c\u4e4e\u5df2\u79bb\u7ebf\u3002\
+<p style="font-size:18px; margin-top: 6%"> \
```
Comment (oleg-nenashev) [inline core/src/main/resources/jenkins/install/pluginSetupWizard_zh_CN.properties]: Wow, it is really being stored in localization files 🤦‍♂️

File: core/src/main/resources/jenkins/install/pluginSetupWizard_zh_CN.properties [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+installWizard_upgradePanel_message=jenkins {0} \u5305\u62ec\u4e00\u4e9b\u4e0d\u9519\u7684\u65b0\u7279\u6027\uff0c\u6211\u4eec\u8ba4\u4e3a\u4f60\u53ef\u80fd\u4f1a\u559c\u6b22\uff0c\u5b89\u88c5\u8fd9\u4e9b\u9644\u52a0\u63d2\u4ef6\u6765\u4f53\u9a8c\uff01
+installWizard_upgradePanel_skipRecommendedPlugins=\u4e0d\uff0c\u8c22\u8c22
+installWizard_upgradeComplete_title=\u66f4\u65b0
+installWizard_pluginsInstalled_banner=\u6b22\u8fce\u4f7f\u7528jenkins {0} \uff01
+installWizard_upgradeComplete_message=\u606d\u559c\uff01\u4f60\u5df2\u66f4\u65b0\u5230jenkins {0} \u3002</p><p>\u8bbf\u95ee<a target="_blank" href="https://jenkins.io/2.0/">jenkins\u7f51\u7ad9</a>\u4e86\u89e3\u66f4\u591a\u65b0\u7279\u6027\u7684\u4fe1\u606f\u3002
+installWizard_upgradeSkipped_title=\u66f4\u65b0
+installWizard_upgradeSkipped_banner=\u7279\u6027\u672a\u5b89\u88c5
+installWizard_upgradeSkipped_message=<div class="alert alert-warning fade in">\u5efa\u8bae\u7684\u63d2\u4ef6\u5c06\u4e0d\u88ab\u5b89\u88c5\u3002</div> \
+<p style="padding: 0 4px">\u5982\u679c\u4f60\u6539\u53d8\u4e3b\u610f\u4e86\uff0c\u8fd8\u53ef\u4ee5\u4ece\u63d2\u4ef6\u7ba1\u7406\u5668\u5b89\u88c5\u65b0\u7279\u6027\u3002</p>\
```
Comment (oleg-nenashev) [inline core/src/main/resources/jenkins/install/pluginSetupWizard_zh_CN.properties]: same

**Review by oleg-nenashev — COMMENTED** [review]
From what I see all tags are correct, have not reviewed the text.

CC @kzantow @tfennelly who originally put these HTML tags to messages. IMHO it's a bug, not a feature

File: core/src/main/resources/jenkins/install/pluginSetupWizard_zh_CN.properties (hunk 1 above)
Comment (node) [reply core/src/main/resources/jenkins/install/pluginSetupWizard_zh_CN.properties]: Should I remove all HTML in the property file when translate ?  Include <a> tag ?

Comment (oleg-nenashev) [reply core/src/main/resources/jenkins/install/pluginSetupWizard_zh_CN.properties]: It's not your problem, I'd say. It was introduced by the Setup Wizard developers. I would keep it as is in the current pull request.

**Review by oleg-nenashev — APPROVED** [review]

**Comment by LinuxSuRen** [comment]
do all these commits will squash in the merge progress?
@oleg-nenashev

File: core/src/main/resources/jenkins/install/pluginSetupWizard_zh_CN.properties [hunk 3]
```diff
@@ -21,78 +21,78 @@
...
 
-installWizard_welcomePanel_title=新手入门
-installWizard_welcomePanel_banner=自定义Jenkins
-installWizard_welcomePanel_message=插件通过附加特性来扩展Jenkins以满足不同的需求。
-installWizard_welcomePanel_recommendedActionTitle=安装推荐的插件
-installWizard_welcomePanel_recommendedActionDetails=安装Jenkins社区推荐的插件。
-installWizard_welcomePanel_customizeActionTitle=选择插件来安装
-installWizard_welcomePanel_customizeActionDetails=选择并安装最适合的插件。
+installWizard_welcomePanel_title=\u65b0\u624b\u5165\u95e8
+installWizard_welcomePanel_banner=\u81ea\u5b9a\u4e49jenkins
```
Comment (LinuxSuRen) [inline core/src/main/resources/jenkins/install/pluginSetupWizard_zh_CN.properties]: here 'jenkins' should be 'Jenkins'

Comment (LinuxSuRen) [reply core/src/main/resources/jenkins/install/pluginSetupWizard_zh_CN.properties]: 应该统一是Jenkins，首字母大写的吧

File: core/src/main/resources/jenkins/install/SetupWizard/proxy-configuration_zh_CN.properties [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+HTTP\ Proxy\ Configuration=\u914d\u7f6eHTTP\u4ee3\u7406
```
Comment (LinuxSuRen) [inline core/src/main/resources/jenkins/install/SetupWizard/proxy-configuration_zh_CN.properties]: 个人感觉这里应该是个名词，如果“配置”在前的话则是动词
HTTP代理配置

**Comment by oleg-nenashev** [comment]
@LinuxSuRen I do not squash commits while commits are atomic and have good commit messages

**Comment by oleg-nenashev** [comment]
In this case I have no strong preference

File: core/src/main/resources/jenkins/install/SetupWizard/proxy-configuration_zh_CN.properties (hunk 4 above)
Comment (node) [reply core/src/main/resources/jenkins/install/SetupWizard/proxy-configuration_zh_CN.properties]: 是的，先merge后面我再更新下吧。

Comment (LinuxSuRen) [reply core/src/main/resources/jenkins/install/SetupWizard/proxy-configuration_zh_CN.properties]: ok

**Comment by oleg-nenashev** [comment]
There are update/create cycles for the same entries, so I will squash the PR

**Pull request merged**